src/BrandedNavBar/renderSubMenuItems.tsx

```diff
@@ -67,33 +19,34 @@ const renderSubMenuLink = (subMenuItem, onItemClick) => (
     style={{ whiteSpace: "nowrap" }}
     key={subMenuItem.key ?? subMenuItem.name}
   >
-    <SubMenuLink
-      fontSize="small"
-      lineHeight="smallTextBase"
+    <DropdownLink
       py="half"
       onClick={onItemClick}
       href={subMenuItem.href}
       to={subMenuItem.to}
       as={subMenuItem.as}
     >
       {subMenuItem.name}
-    </SubMenuLink>
+    </DropdownLink>
   </li>
 );
 
 const renderCustom = (subMenuItem, onItemClick) => (
-  <ApplySubMenuLinkStyles
+  <li
+    style={{ whiteSpace: "nowrap" }}
     key={subMenuItem.key ?? subMenuItem.name}
-    onClick={onItemClick}
   >
-    {subMenuItem.render()}
-  </ApplySubMenuLinkStyles>
+    {subMenuItem.render(onItemClick)}
+  </li>
 );
 
 const renderText = (subMenuItem) => (
-  <SubMenuText key={subMenuItem.key ?? subMenuItem.name}>
-    {subMenuItem.name}
-  </SubMenuText>
+  <li
+    style={{ whiteSpace: "nowrap" }}
```

Comment:
I think we should extract this to a styled component. Inline styles can make this very hard to override.